### PR TITLE
Add documentation about the `docker-pre-stop-print` script

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: docs/src/main/resources/templates/docker.html [hunk 1]
```diff
@@ -104,3 +104,25 @@ <h4 id="docker_sentry">
...
   <a href="https://docs.sentry.io/clients/java/config/">https://docs.sentry.io/clients/java/config/</a>
 </p>
+
+<h4 id="stop">
+  Graceful stop
+  <a class="headerlink" href="#stop" title="Permalink to this headline">¶</a>
+</h4>
+
+<p>
+  The container can be stopped gracefully by calling the <code>docker-pre-stop-print</code> scrypt. This
```
Comment (sebr72) [inline docs/src/main/resources/templates/docker.html:114]: scrypt = script

File: docs/src/main/resources/templates/docker.html [hunk 2]
```diff
@@ -104,3 +104,25 @@ <h4 id="docker_sentry">
...
+  script will trigger a graceful stop, by stopping starting new jobs and waiting for the current jobs to
+  finish.
+</p>
+
+<p>In Kubernetes you can set it as a liveness probe</p>
```
Comment (sebr72) [inline docs/src/main/resources/templates/docker.html:119]: Health status is a better liveness probe. This script does not help to determine when to restart a container. It actually shuts it down.

**Review by sebr72 — CHANGES_REQUESTED** [review]

File: docs/src/main/resources/templates/docker.html (hunk 2 above)
Comment (sbrunner) [reply docs/src/main/resources/templates/docker.html]: liveness probe => preStop hook :-)

**Comment by sebr72** [comment]
@sbrunner Thanks looks good.

**Pull request merged**